### ISSUE TITLE
Should not scan procfs / sysfs

### ISSUE THREAD

**Issue by JustAPerson** [sflanaga/du2#3]
I ran `du2 /` to scan my whole computer

My results were a little skewed
```
Top largest file(s): 10
       127  TB /proc/kcore
```

Perhaps `du2` shouldn't traverse `/proc` or `/sys` since these are just windows into the kernel state, and not actual data stored on a drive.

**Comment by sflanaga** [comment]
yeah... me too.  thx for putting this here to remind me to give that issue some love.

**Comment by sflanaga** [comment]
In the interim, ` du2 / --exclude-re '^/proc/.*|^/sys/.*' `  is reasonable work around.
